Quantum-Control-Applications/Superconducting/Two-Fixed-Coupled-Transmons/Use_Case_1_Two_qubit_RB_without_baking/README.md

```diff
@@ -0,0 +1,138 @@
+# Two-qubit RB generated by cross-resonance CNOTs without pre-baking
+
+_Authors and experimenters: Ziwen Huang, Paul Heidler, Nicholas Bornman_
+
+*Important note: the code in this folder is the code that was used for performing two-qubit randomized benchmarking with cross-resonance*
+*gates for a pair of coupled transmon qubits.*
+
+The file for direct use is [run_two_qubit_rb_CR_CNOT.py](run_two_qubit_rb_CR_CNOT.py), which relies on random Cliffords generation 
+coded in [TwoQ_RB_Sequence_Generation_CNOT_CR.py](TwoQ_RB_Sequence_Generation_CNOT_CR.py).
+
+## 1. Pre-requisite calibrations and some notes
+
+* Calibrate readout pulses and optimize state discrimination.
+* Calibrate and optimize qubit XY pulses. Single qubit operations are based on x90, -x90, x180, y90, -y90, and y180.
+* Our two qubit RB is based on CNOT as the entangling operation which is implemented by direct CR gate. Replace the pertinent lines in the definition of the function `play_sequence` in [run_two_qubit_rb_CR_CNOT.py](run_two_qubit_rb_CR_CNOT.py) (switch case 49) if echoed CR is to be used.
+* No baking will be directly called by the user. The user can easily customize their gates, especially the two-qubit gate pulses.
+* Running gate sequences other than those for RB is also allowed if one manually set the `sequence_list`. (See ##### TEST EXAMPLE in [run_two_qubit_rb_CR_CNOT.py](run_two_qubit_rb_CR_CNOT.py).)
```

Comment:
(See example in [run_two_qubit_rb_CR_CNOT.py](run_two_qubit_rb_CR_CNOT.py).)

Reply:
correcting myself: 
(See TEST EXAMPLE in [run_two_qubit_rb_CR_CNOT.py](https://github.com/qua-platform/qua-libs/pull/run_two_qubit_rb_CR_CNOT.py).)